README.md

```diff
@@ -136,34 +100,64 @@ Stack: PSP
 ++ MbedOS Error Info ++
 Error Status: 0x80FF013D Code: 317 Module: 255
 Error Message: Fault exception
-Location: 0x5F3F
-Error Value: 0x5CB8
-Current Thread: main  Id: 0x20001FC0 Entry: 0x64F9 StackSize: 0x1000 StackMem: 0x20000FC0 SP: 0x2002FF70
-For more info, visit: https://armmbed.github.io/mbedos-error/?error=0x80FF013D
--- MbedOS Error Info -
-== Your last reboot was triggered by an error, below is the error information ==
-
-++ MbedOS Error Info ++
-Error Status: 0x80FF013D Code: 317 Module: 255
-Error Message: System rebooted due to fatal error
-Location: 0x5F3F
-File: mbed_error.c+211
-Error Value: 0x5CB8
-Current Thread: main  Id: 0x20001FC0 Entry: 0x64F9 StackSize: 0x1000 StackMem: 0x20000FC0 SP: 0x2002FF70
-For more info, visit: https://armmbed.github.io/mbedos-error/?error=0x80FF013D
+Location: 0x178C
+Error Value: 0x1FFF0400
+Current Thread: main Id: 0x20000E68 Entry: 0x319B StackSize: 0x1000 StackMem: 0x20001E60 SP: 0x20002E50 
+For more info, visit: https://mbed.com/s/error?error=0x80FF013D&tgt=K64F
 -- MbedOS Error Info --
 
-== Reboot count(=1) exceeded maximum, system halting ==
-Reboor error callback received
+= System will be rebooted due to a fatal error =
+= Reboot count(=1) reached maximum, system will halt after rebooting =
+ Error:
+Status: 0x80FF013D
+Value: 0x1FFF0400
+Address: 0x178C
+Reboot count: 0x1
+CRC: 0x958A47DB
+This is the crash reporting Mbed OS example
+0: 0xDE0C
+1: 0x0
+2: 0xE000ED00
+3: 0xAAA3
+4: 0x0
+5: 0x0
+6: 0x0
+7: 0x0
+8: 0x0
+9: 0x0
+10: 0x0
+11: 0x0
+12: 0x1FFF052C
+13: 0x20002E50
+14: 0x1817
+15: 0x178C
+```
+## Crash-report platform configuration
 
+The crash report feature can be enabled by `platform.crash-capture-enabled: true` configuration in `mbed_app.json` as below:
```

Comment:
Say that this is enabled on some targets by default. I assume that if it is not enabled then the loader/sactter file should be modified, in which case you could provide a link to explain what needs to be done.